scss/_settings.scss

```diff
@@ -51,14 +51,14 @@ $fd-type--base: 14 !default;
 $fd-type: () !default;
 $fd-type: map-merge((
   //"key": font-size line-height margin-bottom
-  "-1": #{12/$fd-type--base}rem 16/12 8px, //12/16 8 btm
-  "0": #{14/$fd-type--base}rem 20/14 12px, //14/20 12
-  "1": #{16/$fd-type--base}rem 20/16 12px, //16/20 12
-  "2": #{18/$fd-type--base}rem 24/18 12px, //18/24 12
-  "3": #{20/$fd-type--base}rem 28/20 12px, //20/28 12
-  "4": #{24/$fd-type--base}rem 32/24 16px, //23/32 16
-  "5": #{36/$fd-type--base}rem 40/36 20px, //36/40 20
-  "6": #{48/$fd-type--base}rem 52/48 24px, //48/52 24
+  "-1": 12px 16/12 8px, //12/16 8 btm
+  "0": 14px 20/14 12px, //14/20 12
+  "1": 16px 20/16 12px, //16/20 12
+  "2": 18px 24/18 12px, //18/24 12
+  "3": 20px 28/20 12px, //20/28 12
+  "4": 24px 32/24 16px, //23/32 16
+  "5": 36px 40/36 20px, //36/40 20
+  "6": 48px 52/48 24px, //48/52 24
```

Comment:
These were doing calculations to figure out the `rem` value, but were using the `px` value to do the math.  Without a specified default font size on the `<body>`, we need to specify all this directly on the components.